tests/Common/ComponentsTest.php

```diff
@@ -256,7 +256,7 @@ public function testConfigurationNameShouldBeRequired()
             $this->fail('Params should be invalid');
         } catch (\Keboola\StorageApi\ClientException $e) {
             $this->assertEquals('storage.components.validation', $e->getStringCode());
-            $this->assertContains('name', $e->getMessage());
+            $this->assertContains('name', $e->getContextParams()['errors'][0]['key']);
```

Comment:
Myslím, že by ta error message měla taky zůstat stejná (někdo podle ní může dělat try/catch). A může to mít navíc ten context...

Reply:
Upravil som to aby vracalo staru message